File: Simple programs/test1.c
#include "mpi.h"
#include "stdio.h"

int main (int argc, char** argv) {
  int N, Rank, tmp;
  MPI_Status status;
  MPI_Init(&argc, &argv);
  MPI_Comm_size(MPI_COMM_WORLD, &N);
  MPI_Comm_rank(MPI_COMM_WORLD, &Rank);
  printf("Hello world from process %i \n", Rank);
  MPI_Finalize();
  return 0;
}

File: Simple programs/test1.f
       program main_mpi
       include 'mpif.h'
       integer myid, numprocs, ierr
       call MPI_Init(ierr)
       call MPI_Comm_rank(MPI_COMM_WORLD, myid, ierr)
       call MPI_Comm_size(MPI_COMM_WORLD, numprocs, ierr)
       print *, "process ", myid, " of ", numprocs
       call MPI_Finalize(ierr)
       end program main_mpi
